client/src/push-notifications/serviceWorker.ts

```diff
@@ -3,16 +3,20 @@ import { ItemWithParsedPayload } from "../types/ItemWithParsedPayload";
 const showNotification = (
   item: ItemWithParsedPayload & {
     clientId?: string;
+    firstName?: string;
+    lastName?: string;
   }
 ) => {
+  const user =
+    item.firstName && item.lastName
+      ? `${item.firstName} ${item.lastName}`
+      : item.userEmail.substring(0, item.userEmail.indexOf("@"));
+
```

Comment:
If this is just for display we could attempt to format the name section of the email with something like 
`item.userEmail..split("@")[0].split(".").map(name  => name.charAt(0).toUpperCase() + name.slice(1)).join(" ")`